capstone/src/main/webapp/index.html

```diff
@@ -25,7 +25,7 @@
           <label for="black-owned-business"> Black-owned Businesses</label><br>
         </label>
         <label class="checkbox-inline">
-          <input type="text" name="search" placeholder="Search Google Maps">
+          <input type="text" id="search" name="search" placeholder="Search Google Maps">
```

Comment:
qq, why the name="search" and id="search" ? I was under the impression you would only need one or the other. It's possible I'm missing something though.

Reply:
good catch-- I mean't to delete the name="search"